### PR TITLE
Navigation drawer + empty battle fragment

### PR DESCRIPTION
**Pull request by rekagiday**
- buildings fragment included, the final version of buildings fragment is on a different branch
- added custom icons
- removed logout button from main activity and added it to navigation drawer menu

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".LoginActivity"></activity>
+        <activity android:name=".LoginActivity" />
+
     </application>
 
-</manifest>
+</manifest>
```
Comment (adamgyulavari) [inline app/src/main/AndroidManifest.xml]: @rekagiday new line at the end of file

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/MainActivity.java [hunk 2]
```diff
@@ -39,29 +49,33 @@ protected void onCreate(Bundle savedInstanceState) {
...
-            public void onClick(View v) {
-                logout();
-            }
-        });
-
         MainFragment mainFragment = new MainFragment();
         FragmentManager fragmentManager = getSupportFragmentManager();
         FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
-        fragmentTransaction.add(R.id.main_activity_layout, mainFragment);
+        fragmentTransaction.add(R.id.flContent, mainFragment);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/MainActivity.java]: @rekagiday ids should have either `camelCase` or `lower_under_score` style naming but use the same, and meaningful names, what is `flContent`

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/MainActivity.java [hunk 3]
```diff
@@ -39,29 +49,33 @@ protected void onCreate(Bundle savedInstanceState) {
...
+
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        DrawerLayout drawer = (DrawerLayout) findViewById(R.id.drawer_layout);
+        ActionBarDrawerToggle toggle = new ActionBarDrawerToggle(
+                this, drawer, toolbar, R.string.navigation_drawer_open, R.string.navigation_drawer_close);
+        drawer.setDrawerListener(toggle);
+        toggle.syncState();
+        NavigationView navigationView = (NavigationView) findViewById(R.id.nvView);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/MainActivity.java]: @rekagiday here as well

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/MainActivity.java (hunk 2 above)
Comment (adamgyulavari) [reply app/src/main/java/com/greenfox/tribesoflagopusandroid/MainActivity.java]: you have a method for doing this, why not call that?

File: app/src/main/res/layout/activity_login.xml [hunk 4]
```diff
@@ -9,7 +9,7 @@
 
     <EditText
         android:id="@+id/usernameText"
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
```
Comment (adamgyulavari) [inline app/src/main/res/layout/activity_login.xml]: @rekagiday why the changes in login?

File: app/src/main/res/layout/fragment_battle.xml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.greenfox.tribesoflagopusandroid.fragments.BattleFragment">
+
+    <ImageView
+        android:id="@+id/imageView2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:srcCompat="@drawable/battle" />
+</FrameLayout>
```
Comment (adamgyulavari) [inline app/src/main/res/layout/fragment_battle.xml]: @rekagiday new line at the end

File: app/src/test/java/com/greenfox/tribesoflagopusandroid/MainActivityTest.java [hunk 6]
```diff
@@ -37,26 +37,4 @@ public void userDoesNotExistAndGoToLoginActivityTest() throws Exception {
...
-
-        main.editor.putString(USERNAME, "testUsername");
-        main.editor.apply();
-
-        Button button = (Button) main.findViewById(R.id.logout);
-        button.performClick();
-
-        assertThat(null, is(main.preferences.getString(USERNAME, null)));
-    }
-
```
Comment (adamgyulavari) [inline app/src/test/java/com/greenfox/tribesoflagopusandroid/MainActivityTest.java]: @rekagiday do not remove these, just test the logout in the navdrawer

Comment (adamgyulavari) [reply app/src/test/java/com/greenfox/tribesoflagopusandroid/MainActivityTest.java]: and also where are the tests? :)

**Review by adamgyulavari — DISMISSED** [review]

File: app/src/main/res/layout/fragment_battle.xml (hunk 5 above)
Comment (rekagiday) [reply app/src/main/res/layout/fragment_battle.xml]: all done

**Review by viktorhegyi — APPROVED** [review]
